Sources/Action/UIKitExtensions/UIAlertAction+Action.swift

```diff
@@ -5,7 +5,7 @@ import RxCocoa
 
 public extension UIAlertAction {
 
-    public static func Action(_ title: String?, style: UIAlertActionStyle) -> UIAlertAction {
+    public static func Action(_ title: String?, style: UIAlertAction.Style) -> UIAlertAction {
```

Comment:
Oh shoot, one thing. Is this backwards compatible with older versions of Swift/Xcode/etc? Can we do something like this?

```swift
#if swift(>=4.2)
  typealias ActionStyle = UIAlertAction.Style // or whatever name
#else
  typealias ActionStyle = UIAlertActionStyle
#endif

public static func Action(_ title: String?, style: ActionStyle) -> UIAlertAction {
```

Sorry I didn't catch that earlier – I'd like to stay backwards compatible for a little while 😄

Reply:
nice catch!